### ISSUE TITLE
how to intercept and handle a wrong request

### ISSUE THREAD

**Issue by nickeljew** [oatpp/oatpp#643]
When incoming request is in bad format, like missing content-type header, the application will crash with segmentation fault, but I handles this case directly in endpoint  Action act() method by myself before it crashes. It that anyway to intercept the handling of incoming request before that? Or catch that error to avoid crash?

**Comment by lganzzzo** [comment]
Hello @nickeljew ,

Please post your endpoint here.

If request reaches the `act()` method - then most probably it's your code that has to take care about `nullptr` and stuff.

**Comment by nickeljew** [comment]
> 

No, it does not reach act() method. I just put a printf line inside act() at the beginning, but no any print-out and crashes at once.
